### PR TITLE
Nixstats incoming webhook script

### PR DESCRIPTION
**Pull request by vfuse**
Receive notifications from nixstats.com in rocket.chat

### CODE REVIEW

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+1. In Rocket.Chat go to "Administration"->"Integrations" and create "New Integration"
+2. Choose Outgoing WebHook
+3. Select **Message Sent** as Event trigger
+3. Enter **ns** as trigger word
+4. Enter **https://api.eu.nixstats.com/v1/** as URLs
+5. Avatar URL **https://nixstats.com/images/favicon.png**
+6. **Token**, this is your nixstats API token, [create an API key](https://nixstats.com/settings/api).
+7. Script Enabled set to **True**
+
+Paste this in javascript in the "Script" textarea on Rocket.Chat webhook settings
```
Comment (geekgonecrazy) [inline 5. Administrator Guides/Integrations/nixstats.md]: > Paste this in javascript in the "Script"

Should be something like:
> Paste this in the "Script" 
or
> Paste this javascript in the "Script"

**Review by geekgonecrazy — DISMISSED** [review]

**Comment by geekgonecrazy** [comment]
@sampaiodiego will the backticks in the script be a problem for the docs renderer?

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+1. In Rocket.Chat go to "Administration"->"Integrations" and create "New Integration"
+2. Choose Incoming WebHook
+3. Follow all instructions like Enable, give it a name, link to channel etc. Set "Enable Script" to true and enter the javascript in the "Script" box
+4. Press Save changes and copy the *Webhook URL* (added just below the script box)
+5. Go to nixstats.com -> Settings -> Notification Contacts -> Add (or Edit a contact) contact
+6. Paste the Rocket.Chat url you've copied in step 4
+
+Paste this in javascript in the "Script" textarea on Rocket.Chat webhook settings
+
+```/* exported Script */
```
Comment (sampaiodiego) [inline 5. Administrator Guides/Integrations/nixstats.md]: please start the code on a new line

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+3. Enter **ns** as trigger word
+4. Enter **https://api.eu.nixstats.com/v1/** as URLs
+5. Avatar URL **https://nixstats.com/images/favicon.png**
+6. **Token**, this is your nixstats API token, [create an API key](https://nixstats.com/settings/api).
+7. Script Enabled set to **True**
+
+Paste this in javascript in the "Script" textarea on Rocket.Chat webhook settings
+
+
+```/* exported Script */
```
Comment (sampaiodiego) [inline 5. Administrator Guides/Integrations/nixstats.md]: please start the code on a new line

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+  }
+}
+```
+
+
+After saving the data you can use the following commands to retrieve data.
+
+`ns servers list` to list your servers with their ID's and load average.
+
+<img src="https://github.com/NIXStats/Nixstats-integrations/blob/master/rocketchat/serverlist.png" data-canonical-src="graphs.png" width="400" />
```
Comment (sampaiodiego) [inline 5. Administrator Guides/Integrations/nixstats.md]: this image does not render

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+`ns graphs [serverid]` to retrieve a graph of Memory, Network, Load average and Disk usage of the specified server.
+
+<img src="https://github.com/NIXStats/Nixstats-integrations/blob/master/rocketchat/graphs.png" data-canonical-src="graphs.png" width="400" />
```
Comment (sampaiodiego) [inline 5. Administrator Guides/Integrations/nixstats.md]: this image does not render

**Review by sampaiodiego — DISMISSED** [review]

File: 5. Administrator Guides/Integrations/nixstats.md (hunk 3 above)
Comment (docsncode) [reply 5. Administrator Guides/Integrations/nixstats.md]: @vfuse could you address @sampaiodiego and then we can get this merged!

File: 5. Administrator Guides/Integrations/nixstats.md (hunk 1 above)
Comment (geekgonecrazy) [inline 5. Administrator Guides/Integrations/nixstats.md]: @vfuse Also this is a small one, but greatly helps readability.

@SeanPackham  :top:

**Review by vfuse — COMMENTED** [review]
Starting code on newline

File: 5. Administrator Guides/Integrations/nixstats.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+Add Nixstats notifications via a new WebHook in Rocket.Chat
+
+1. In Rocket.Chat go to "Administration"->"Integrations" and create "New Integration"
+2. Choose Incoming WebHook
+3. Follow all instructions like Enable, give it a name, link to channel etc. Set "Enable Script" to true and enter the javascript in the "Script" box
+4. Press Save changes and copy the *Webhook URL* (added just below the script box)
+5. Go to nixstats.com -> Settings -> Notification Contacts -> Add (or Edit a contact) contact
+6. Paste the Rocket.Chat url you've copied in step 4
+
+Paste this in javascript in the "Script" textarea on Rocket.Chat webhook settings
```
Comment (geekgonecrazy) [inline 5. Administrator Guides/Integrations/nixstats.md]: Please adjust the wording here to something like:
> Paste this javascript in the "Script" textarea on Rocket.Chat webhook settings

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

**Review by geekgonecrazy — APPROVED** [review]

**Comment by geekgonecrazy** [comment]
Thanks for your contribution @vfuse sorry for so many little changes.

**Pull request merged**